<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanSettings</class>
 <widget class="QWidget" name="ScanSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>扫描设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0">
   <item>
    <widget class="QGroupBox" name="templateGroup">
     <property name="title">
      <string>模板</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <widget class="QLineEdit" name="templatePath"/>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="editTemplatePath">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupSave">
     <property name="title">
      <string>保存</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_11">
      <item row="1" column="1">
       <widget class="QLineEdit" name="outputFormat">
        <property name="text">
         <string>$(Date)/$(Material)/$(Date)-$(Time)-$(Surface)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="neverAskFileName">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;开启后，创建扫查任务时不再弹出文件名修改的窗口&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>不询问文件名</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="dataLevel"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="dataLevelTitle">
        <property name="text">
         <string>存储级别</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="outputFormatTitle">
        <property name="text">
         <string>文件名</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupCustomField">
     <property name="title">
      <string>报表字段</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTreeView" name="editField"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="addField">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../../Resources/Resources.qrc">
            <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="deleteField">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../../Resources/Resources.qrc">
            <normaloff>:/icons/minus.png</normaloff>:/icons/minus.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupAdvanced">
     <property name="title">
      <string>高级</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="scanWithoutDatum">
        <property name="text">
         <string>未复位时允许扫描 (危险)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showLegacyScanPlan">
        <property name="text">
         <string>显示旧版本扫描计划(软件重启后生效)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
